src/hotspot/os/windows/os_windows.cpp

```diff
@@ -5103,9 +5104,6 @@ char* os::pd_map_memory(int fd, const char* file_name, size_t file_offset,
       return nullptr;
     }
 
-    // Record virtual memory allocation
-    MemTracker::record_virtual_memory_reserve_and_commit((address)addr, bytes, CALLER_PC);
-
```

Comment:
Why is this removed?

Reply:
We found this call duplicated, since the `MemTracker::record_..._and_commit` is called inside the `os::map_memory` after `pd_map_memory` is called.
Here, the requested address is used, but there the result address is used.